### ISSUE TITLE
code error:  tot += F.ross_entropy(pred.unsqueeze(dim=0), true_mask.unsqueeze(dim=0)).item()

### ISSUE THREAD

**Issue by XUYUNYUN666** [milesial/Pytorch-UNet#108]
there are errrors about the dimensions,  the true_mask dont need to be unsqueezed.

**Comment by milesial** [comment]
By the [documentation](https://pytorch.org/docs/stable/nn.functional.html?highlight=cross_entropy#torch.nn.functional.cross_entropy) the first dimensions is the batch dimension
